fog/ledger/server/src/db_fetcher.rs

```diff
@@ -116,7 +116,7 @@ impl<DB: Ledger, E: LedgerEnclaveProxy + Clone + Send + Sync + 'static> DbFetche
                 log::info!(self.logger, "Db fetcher thread stop requested.");
                 break;
             }
-            self.next_block_index = 0;
+            self.next_block_index = 1;
```

Comment:
So this means we don't scan block 0? Is this the same as what we do elsewhere?

Reply:
The commit message says we're fixing something with this change but I don't understand

Reply:
can you write what the reason was that we're setting this to 1 and not 0, ideally in a code comment, or at least in the review comments?

I don't remember what the outcome of this was anymore